python/pyspark/sql/connect/functions/builtin.py

```diff
@@ -3804,12 +3804,14 @@ def sha2(col: "ColumnOrName", numBits: int) -> Column:
 sha2.__doc__ = pysparkfuncs.sha2.__doc__
 
 
-def hll_sketch_agg(col: "ColumnOrName", lgConfigK: Optional[Union[int, Column]] = None) -> Column:
+def hll_sketch_agg(
+    col: "ColumnOrName",
+    lgConfigK: Optional[Union[int, Column]] = None,
+) -> Column:
```

Comment:
This doesn't look like a doc string. May I ask if we need this style change?

Reply:
this line is kind of long (to my screen), so I break it to multiple lines:

<img width="872" alt="image" src="https://github.com/apache/spark/assets/7322292/86029dd9-9939-4b14-8df2-fc9c89feffdb">